roles/sap_hana_install/tasks/pre_install.yml

```diff
@@ -63,7 +54,16 @@
       register: __sap_hana_install_register_stat_software_extract_directory
       failed_when: no
 
-# In case more than on installation is ongoing and extracting to the same shared directory, wait until the extraction has completed:
+    - name: SAP HANA Pre Install - Change ownership of software extract directory '{{ sap_hana_install_software_extract_directory }}'
+      ansible.builtin.file:
+        path: "{{ sap_hana_install_software_extract_directory }}"
+        state: directory
+        recurse: yes
+        mode: '0755'
+        owner: root
+        group: root
+
+# In case more than one installation is ongoing and extracting to the same shared directory, wait until the extraction has completed:
```

Comment:
Needs to be removed. We are checking if the extract directory exists. If it exists, we will later search for a directory named `SAP_HANA_DATABASE`. If it does not exist or if it exists are there is no directory named `SAP_HANA_DATABASE` in it, we will remove and recreate the directory with mode 0755. This happens in  tasks/pre_install/hdblcm_prepare.yml